src/Fozzie.java

```diff
@@ -1,11 +1,94 @@
 package fozzie;
 
-import fozzie.CommandFactory;
+import fozzie.fuzzer.CommandFactory;
+
+import java.io.*;
+import java.net.*;
+import java.lang.*;
 
 public class Fozzie {
+	
+	private static final int DRONE_PORT = 5556;
+	private static final int RECEIVE_PORT = 5554;
+	private static final int DRONE_NAVDATA_PORT = 5554;
+	private static final String DRONE_IP_ADDRESS = "192.168.1.1";
+  private static final String SAFE_MODE = "AT*PCMD=1,0,0,0,0,0\r"; // PCMD hovering mode
```

Comment:
IMO this should be "AT*REF=1,290717696\rAT*PCMD=1,0,0,0,0,0\r" - including the ref command is better parity with the app, and counteracts bad ref commands.